bpf/bpf_lxc.c

```diff
@@ -128,6 +128,7 @@ static inline int ipv6_l3_from_lxc(struct __sk_buff *skb,
 	void *data, *data_end;
 	union v6addr *daddr, orig_dip;
 	bool orig_was_proxy;
+        uint16_t dstID = WORLD_ID;
```

Comment:
Minor style thing here: the `bpf/` code follows linux kernel coding style, ie tabs rather than spaces.

Reply:
will do.